benchmarks/README.md

```diff
@@ -0,0 +1,74 @@
+# README
+
+## Overview
+We use `asv` to benchmark some representative brainglobe workflows. The `asv` workflow is roughly as follows:
+1. `asv` creates a virtual environment (as defined in the `asv.conf.json` file).
+1. It installs the version of the `brainglobe-workflows` software package corresponding to the tip of the locally checked-out branch.
+1. It runs the benchmarks defined (locally) under `brainglobe-workflows/benchmarks/benchmarks` and saves the results to `brainglobe-workflows/benchmarks/results` as json files.
+1. With `asv publish`, the output json files are 'published' into an html directory (`brainglobe-workflows/benchmarks/html`)
+1. With `asv preview` the html directory can be visualised in a static site.
+
+
+There are three main ways in which these benchmarks can be useful to developers:
+1. Developers can run the available benchmarks locally [on a small test dataset](#running-benchmarks-locally-on-default-small-dataset).
+1. Developers can run these benchmarks on [data they have stored locally](#running-benchmarks-locally-on-custom-data).
+1. We also plan to run the benchmarks internally on a large dataset, and make the results publicly available.
+
+## Installation
```

Comment:
This section seems to say:
- Install asv
- But you didn't need to install asv, as we're doing it for you. 

Is it worth simplifying this to just present a single way of doing things?